cmd/rookflex/cmd/unmount.go

```diff
@@ -53,7 +54,8 @@ func handleUnmount(cmd *cobra.Command, args []string) error {
 	mounter := getMounter()
 
 	// Check if it's a cephfs
-	err = executor.ExecuteCommand(false, "", "df", "--type", cephFS, mountDir)
+	cmd := exec.Command("df", "--type", cephFS, mountDir)
```

Comment:
are there more instances in the flexvolume binary that should be similarly updated?  Should `ExecuteCommand` be updated to optionally not log the output? https://github.com/rook/rook/blob/master/pkg/util/exec/exec.go#L58

Reply:
this is the only place in rook flex driver :)
change executor for this single use case is an overkill to me